tests/integration/ceph_csi_test.go

```diff
@@ -44,6 +44,12 @@ func runCephCSIE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suit
 		createCSIStorageClass(k8sh, s, namespace)
 		createCSIRBDTestPod(k8sh, s, namespace)
 		createCSICephFSTestPod(k8sh, s, namespace)
+
+		//cleanup resources created
+		deleteCephCSISecret(k8sh, namespace)
+		deleteCephPools(helper, namespace)
+		deleteCSIStorageClass(k8sh, namespace)
+
```

Comment:
Could you also delete the test rbd and test cephfs pods?

Reply:
done deleted rbd and cephfs pvc and pod